src/client/components/leaderboard/leaderboard.js

```diff
@@ -20,33 +20,36 @@ class Leaderboard extends React.Component {
     return (
       <Card>
         <CardHeader>Statistics</CardHeader>
-          <Table size="sm">
-            <thead>
-              <tr>
-                <th>#</th>
-                <th>Name</th>
-                <th>Passed</th>
-                <th>Card</th>
-                <th>Score</th>
+        <Table size="sm">
+          <thead>
+            <tr>
+              <th>Name</th>
+              <th></th>
```

Comment:
You added this empty column such that the `(you)` does not get cut off, right?

But this still leads to strangely cut off player names

![Screenshot from 2021-07-27 14-44-57](https://user-images.githubusercontent.com/10673021/127155774-d30e812a-31da-4841-9931-4ab45f7e135f.png)

I would propose the following:
1. remove the empty column
2. add the `(you)` in a new line (e.g. by wrapping it in a `div`).

```js
<td>
  <div className={parseInt(this.props.playerID) === idx ? "ownPlayername" : "otherPlayername"}>
    {this.props.names[idx]}
  </div>
  {parseInt(this.props.playerID) === idx && <div><strong>(you)</strong></div>}
</td>
```

Then it would look like

![Screenshot from 2021-07-27 14-45-13](https://user-images.githubusercontent.com/10673021/127156192-4ee8cb3a-53bd-4892-bcec-ff5a1ab7b504.png)